nbgrader/exchange/ngshare/release_feedback.py

```diff
@@ -95,16 +83,13 @@ def post_feedback(self, student_id, timestamp, feedback_info):
         represented as a dictionary with a "path" to the local feedback file and
         "notebook_id" of the corresponding notebook.
         """
-        url = '{}{}/feedback/{}/{}/{}'.format(
-            self.ngshare_url, self.prefix, self.coursedir.course_id,
-            self.coursedir.assignment_id, student_id)
+        url = '/feedback/{}/{}/{}'.format(self.coursedir.course_id, self.coursedir.assignment_id, student_id)
         files = json.dumps([self.encode_file(x['path'],
                                              '{}.html'.format(x['notebook_id'])
                                              ) for x in feedback_info])
-        data = {'user': self.username, 'timestamp': timestamp, 'files': files}
+        data = {'timestamp': timestamp, 'files': files}
 
-        response = requests.post(url, data=data)
-        self.check_response(response)
+        self.ngshare_api_post(url, data)
```

Comment:
Capture return value here and change try-except block to if-statement in lines 72-77.

Reply:
Fixed.